src/main/java/build/buildfarm/common/s3/BUILD

```diff
@@ -0,0 +1,12 @@
+java_library(
+    name = "S3Bucket",
+    srcs = ["S3Bucket.java"],
+    visibility = ["//visibility:public"],
+    deps = [
+        "//src/main/protobuf:build_buildfarm_v1test_buildfarm_java_proto",
+        "@maven//:com_amazonaws_aws_java_sdk_core",
+        "@maven//:com_amazonaws_aws_java_sdk_s3",
+        "@maven//:com_amazonaws_aws_java_sdk_secretsmanager",
+        "@maven//:com_fasterxml_jackson_core_jackson_databind",
+    ],
+)
```

Comment:
Leaving this dangling without any usage, even a test, makes me a little worried for its health.